Build failure: moon-buggy

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

nix build .#moon-buggy

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
 nix build .#moon-buggy
error: Cannot build '/nix/store/8vs7qw8g2dpdkfdw9kcyvrcljcmvsvkb-moon-buggy-1.0.51.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/38f3lnfi5sb5xlhfqyimfysimsf4czzz-moon-buggy-1.0.51
       Last 25 log lines:
       > signal.c:64:1: note: 'generic_handler' declared here
       >    64 | generic_handler (int signum)
       >       | ^~~~~~~~~~~~~~~
       > signal.c: In function 'tstp_handler':
       > signal.c:126:28: error: passing argument 2 of 'install_signal' from incompatible pointer type [-Wincompatible-pointer-types]
       >   126 |   install_signal (SIGTSTP, SIG_DFL);
       >       |                            ^~~~~~~
       >       |                            |
       >       |                            void (*)(int)
       > signal.c:50:42: note: expected 'int (*)(void)' but argument is of type 'void (*)(int)'
       >    50 | install_signal (int signum, RETSIGTYPE (*handler) ())
       > signal.c: In function 'cont_handler':
       > signal.c:133:28: error: passing argument 2 of 'install_signal' from incompatible pointer type [-Wincompatible-pointer-types]
       >   133 |   install_signal (SIGTSTP, tstp_handler);
       >       |                            ^~~~~~~~~~~~
       >       |                            |
       >       |                            void (*)(int)
       > signal.c:50:42: note: expected 'int (*)(void)' but argument is of type 'void (*)(int)'
       >    50 | install_signal (int signum, RETSIGTYPE (*handler) ())
       > signal.c:121:1: note: 'tstp_handler' declared here
       >   121 | tstp_handler (int signum)
       >       | ^~~~~~~~~~~~
       > make[1]: *** [Makefile:333: signal.o] Error 1
       > make[1]: Leaving directory '/build/moon-buggy-1.0.51'
       > make: *** [Makefile:215: all] Error 2
       For full logs, run:
         nix log /nix/store/8vs7qw8g2dpdkfdw9kcyvrcljcmvsvkb-moon-buggy-1.0.51.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.18.2, NixOS, 26.05 (Yarara), 26.05.20251230.e77214d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2+1`
 - nixpkgs: `/nix/store/yjvrn1vnnjz1lkpmifxjli4v14i9zlyh-source`

### Notify maintainers

@rybern

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/mo/moon-buggy/package.nix
+++ b/pkgs/by-name/mo/moon-buggy/package.nix
@@ -6,15 +6,15 @@
 }:
 stdenv.mkDerivation rec {
   pname = "moon-buggy";
-  version = "1.0.51";
+  version = "1.1.0";
 
   buildInputs = [
     ncurses
   ];
 
   src = fetchurl {
-    url = "http://m.seehuhn.de/programs/moon-buggy-${version}.tar.gz";
-    sha256 = "0gyjwlpx0sd728dwwi7pwks4zfdy9rm1w1xbhwg6zip4r9nc2b9m";
+    url = "http://www.seehuhn.de/programs/moon-buggy/moon-buggy-${version}.tar.gz";
+    sha256 = "sha256-JZrm57GDjEBTKvXA8gzXxhc81cVS7eQIEUBkR1vPxbY=";
   };
 
   meta = {

